benchmark/misc/destructuring-bench.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  method: ['old', 'new'],
+  millions: [100]
+});
+
+function runSwapOld(n) {
+  var i = 0, x, y, r;
+  bench.start();
+  for (; i < n; i++) {
+    x = 1, y = 2;
+    r = x;
+    x = y;
+    y = r;
+    assert.strictEqual(x, 2);
+    assert.strictEqual(y, 1);
+  }
+  bench.end(n / 1e6);
+}
+
+function runSwapNew(n) {
```

Comment:
Perhaps this should be named something like `runSwapDestructure` or something and the other one like `runSwapManual`.